### PR TITLE
Add WNP 98 for EU (VPN) (Fixes #11210)

### LINKED ISSUE #11210: [Epic] WNP 98 (EU VPN)
## Description

Launching **March 8** (strings need to be in Pontoon by **March 1st**).

[Launch plan](https://docs.google.com/document/d/1emFAlQB1U9KY--tYGHK3NUvZfTnQZnbcFnTZOwt7Kvg/edit?usp=sharing) (Mozilla access only)

- [x] Page design and content - re-used from previous WNP. Localized. @slightlyoffbeat will provide
- [x] Localized into three locales (see launch plan).
- [x] Geo target these pages to new countries (see launch plan).
- [ ] Copy needs to be in Pontoon by **March 1st**.

Also depends on:

- [ ] https://github.com/mozilla/bedrock/issues/11213

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
Adds a geo-specific WNP for FF 98 promoting Mozilla VPN.

## Issue / Bugzilla link
#11210

## Testing
- https://www-demo1.allizom.org/en-US/firefox/98.0/whatsnew/?geo=se
- https://www-demo1.allizom.org/en-US/firefox/98.0/whatsnew/?geo=fi

### CODE REVIEW

**Comment by alexgibson** [comment]
This has been approved and is ready for review.

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx98-vpn-eu.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      <div class="wnp-main-image">
+        {% include 'firefox/whatsnew/includes/fx98-vpn-eu/vpn-header.svg' %}
+      </div>
+      <div class="mzp-l-content mzp-t-content-md">
+        <h2 class="wnp-main-title">{{ ftl('whatsnew-98-vpn-eu-title') }}</h2>
+        <p class="wnp-main-tagline">{{ ftl('whatsnew-98-vpn-eu-desc') }}</p>
+
+        <div class="wnp-main-cta qa-main-cta-vpn">
+          {{ vpn_product_referral_link(
+              referral_id='whatsnew-98-vpn-eu',
```
Comment (maureenlholland) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx98-vpn-eu.html:35]: ✅ GA cookie set, VPN subscription links look good

> ?plan=price_1J0Y12Kb9q6OnNsL4SB2hhmp&entrypoint=www.mozilla.org-whatsnew&form_type=button&service=e6eb0d1e856335fc&data_cta_position=pricing&utm_source=www.mozilla.org-whatsnew&utm_medium=referral&**utm_campaign=whatsnew-98-vpn-eu**&device_id=5d8154e268cd4738b8335fb1da36d2fd&flow_begin_time=1645441629816&flow_id=ea8a34fdf93fceed1ef43f1b452e4df749ea035fb2f9d1cf46415ebb8a6108f9

File: l10n/configs/pontoon.toml [hunk 2]
```diff
@@ -217,6 +217,13 @@ locales = [
 [[paths]]
     reference = "en/firefox/whatsnew/whatsnew.ftl"
     l10n = "{locale}/firefox/whatsnew/whatsnew.ftl"
+[[paths]]
+    reference = "en/firefox/whatsnew/whatsnew-fx98-vpn-eu.html"
+    l10n = "{locale}/firefox/whatsnew/whatsnew-fx98-vpn-eu.html"
+    locales = [
```
Comment (maureenlholland) [inline l10n/configs/pontoon.toml:223]: **question (non-blocking, curiousity):** what's the best way to confirm the file will show up for these locales in pontoon? checking they are part of the [PROD_LANGUAGES settings](https://github.com/mozilla/bedrock/blob/master/bedrock/settings/base.py#L129)?

**Review by maureenlholland — CHANGES_REQUESTED** [review]
Looks great! One minor blocking issue with the duplicate SVG ids. Keeping those SVGs visible in dark mode is tricky enough so no need for further optimization if it blows away the class names needed for styling.

File: bedrock/firefox/templates/firefox/whatsnew/includes/fx98-vpn-eu/heart.svg [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 480 480">
+<style>.st14{fill-rule:evenodd;clip-rule:evenodd}.st16{fill:#a883f8}.st18,.st19{fill:none;stroke:#fff;stroke-width:4;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10}.st19{stroke:#000}.st20{fill:#ea5e65}</style>
+<g id="Layer_1">
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/whatsnew/includes/fx98-vpn-eu/heart.svg:3]: @maureenlholland Hmm, I thought SVGs use their own namespace and are treated a bit like separate documents (in a similar way to iframes). I can update the IDs easy enough, but is there a specific issue this is solving?

File: l10n/configs/pontoon.toml (hunk 2 above)
Comment (alexgibson) [reply l10n/configs/pontoon.toml:223]: `PROD_LANGUAGES` is a list of all the supported language codes we use in production, but this specific `locales` list here is what gets used to ensure only a subset of those locales appears in Pontoon.

File: bedrock/firefox/templates/firefox/whatsnew/includes/fx98-vpn-eu/heart.svg (hunk 3 above)
[the remark this answers is not in the document]
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/whatsnew/includes/fx98-vpn-eu/heart.svg:3]: Huh, looks like https://validator.w3.org does still call them out as duplicate, so I guess I'll update 👍

**Comment by alexgibson** [comment]
@maureenlholland thanks, updated!

**Review by maureenlholland — APPROVED** [review]
r+ 👍

**Pull request merged**